typing/ikinds/ldd.mli

```diff
@@ -22,16 +22,16 @@ module Make (V : ORDERED) : sig
 
   val new_var : unit -> var
 
-  val var : var -> node
+  val mk_var : var -> node
```

Comment:
`mk_var` and `new_var` sound like they do the same thing. Maybe a name like `node_of_var` would be better than `mk_var`?